README.md

```diff
@@ -80,88 +80,86 @@ interface CheckerInfoMessage {
-    runId: number;
-    method: string;
-    address: string;
-    serviceId: number;
-    serviceName: string;
-    teamId: number;
-    teamName: string;
-    roundId: number;
-    relatedRoundId: number;
-    flag: string | null;
-    flagIndex: number;
-    timeout: number;                            // Timeout in miliseconds
-    roundLength: number;                        // Round Length in seconds
+    taskId: number;                             // The per-ctf unique id of a task.
+    method: string;                             // "putflag", "getflag", "putnoise", "getnoise" or "havoc".
+    address: string;                            // The address of the target team's vulnbox. Can be either an IP address or a valid hostname.
+    teamId: number;                             // The id of the target team.
+    teamName: string;                           // The name of the target team.
+    currentRoundId: number;                     // The id of the current round.
+    relatedRoundId: number;                     // The id of the round in which the "putflag", "putnoise" or "havoc" happened.
+    flag: string | null;                        // The flag for putflag and getflag, otherwise null.
+    variantId: number;                          // The variant id of the task. Used to support different flag and noise methods.
+    timeout: number;                            // Timeout in milliseconds.
+    roundLength: number;                        // Round length in milliseconds.
+    taskContextId: string;                      // The unique identifier of a set of related tasks (i.e. putflag and its getflags, and putnoise and its getnoises, and individual havocs.). Always composed in the following way: "{flag|noise|havoc_s{serviceId}_r{roundId}_t{teamId}_i{index}".
 }
 ```
 Response:
 ```ts
 interface CheckerResultMessage {
-    result: string;                             // "INTERNAL_ERROR", "OK", MUMBLE", or "OFFLINE"
+    result: string;                             // "INTERNAL_ERROR", "OK", MUMBLE", or "OFFLINE".
     message: string | null;
 }
 ```
 
 ## Scoreboard API
 ```ts
 interface ScoreboardInfo {
-    dnsSuffix: string | null;                   // ".bambi.ovh"
+    dnsSuffix: string | null;                   // The DNS suffix, if DNS is used. Example: ".bambi.ovh"
     services: Service[];
     teams: ScoreboardInfoTeam[];
 }
 
 interface ScoreboardInfoTeam {
-    id: number;                                 // 40
-    name: string;                               // "teamname40"
-    logoUrl: string | null;                     // "http://..."
-    flagUrl: string | null;                     // "http://..."
+    id: number;                                 // The id of the team.
+    name: string;                               // The name of the team.
+    logoUrl: string | null;                     // An URL with the team's logo, or null.
+    countryCode: string | null;                 // The ISO 3166-1 alpha-2 country code, or null.
```

Comment:
is that always uppercase, lowercase or not specified? might be relevant when including this directly in image URLs

Reply:
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 ISO says always uppercase

Reply:
might still be worth mentioning that explicitly in the readme